source/_data/glossary.yml

```diff
@@ -21,7 +21,7 @@
 - topic: Discovery
   description: "[Discovery](/integrations/discovery/) is the automatic setup of zeroconf/mDNS and uPnP devices after they are discovered."
 - topic: Domain
-  description: "Entities and services belong to a domain, which is the first part of the entity or service, before the `.`. For example `light.kitchen` is an entity in the `light` domain, while `homeassistant.turn_on` is the `turn_on` service for the `homeassistant` domain."
+  description: "Domains provide generic interfaces for high-level concepts such as `light`, `switch` or `sensor`. These are then implemented by `integrations`. Entities and services belong to a domain, which is the first part of the entity or service, before the `.`. For example `light.kitchen` is an entity in the `light` domain, while `homeassistant.turn_on` is the `turn_on` service for the `homeassistant` domain."
```

Comment:
This is not fully correct, might be confusing.

A domain refers to an integration, each integration has its own domain. They do not only refer to high-level concepts but an integration. For example, the `hue.activate_scene` service is a good example of such a thing.

Reply:
Ooh so that's why it's called domain

Reply:
Okay I've reverted that change

Reply:
How about:

```
Each integration in Home Assitant has a unique identifier: a domain. All of the entities and services available in Home Assistant are provided by integrations and thus belong to such a domain. The first part of the entity or service, before the `.` shows the domain they belong to. For example `light.kitchen` is an entity in the `light` domain from the [light integration](/integrations/light), while `hue.activate_scene` is the `activate_scene` service for the `hue` domain which belongs to the [Hue integration](/integrations/hue).
```

Would that help?

Reply:
Sounds good!